kodilla-spring-web/settings.gradle

```diff
@@ -0,0 +1 @@
+rootProject.name = 'kodilla-spring-web'
```

Comment:
to jest zbedne - settings.gradle musi byc tylko w roocie. 

W roocie brakuje wpisu o wym module
`include 'kodilla-spring-web'`